pvlib/modelchain.py

```diff
@@ -645,9 +650,10 @@ def infer_dc_model(self):
             return self.pvwatts_dc, 'pvwatts'
         else:
             raise ValueError('could not infer DC model from '
-                             'system.module_parameters. Check '
-                             'system.module_parameters or explicitly '
-                             'set the model with the dc_model kwarg.')
+                             'system.arrays[i].module_parameters. Check '
+                             'system.arrays[i].module_parameters or '
```

Comment:
I don't know if this notation will make sense to some users. I'm fine with it, but here's an alternative:

Could not infer DC model from the module_parameters attributes of system.arrays. Check the module_parameters attributes or explicitly set the model with the dc_model keyword argument.